documentation/docs/getting-started/index.md

```diff
@@ -5,149 +5,121 @@ path: /getting-started/
 
 # Getting started
 
-Fusion.js is a web application framework for building high quality universal applications. It provides a modular architecture with a strong focus on testability and maintainability.
+Fusion.js, Uber’s open source universal web framework, is designed to make web development easier and produce lightweight,
+high-performing apps. We initially [built Fusion.js](https://eng.uber.com/fusionjs/) to revamp our own websites, and have
+since offered it to the community as an open source project.
 
-Here are the features you'll find in Fusion.js:
+In this tutorial, we walkthrough how to get started with Fusion.js and have a 'Hello World!' application up and running
+in 5 minutes. The tutorial walks through three easy steps, from creating the application to displaying 'Hello World!' in
+the browser. The 'Hello World!' application is a good starting point for those seeking to get deeper into the Fusion.js
+framework.
 
-- server side rendering and async rendering
-- ES2017 and JSX support out of the box
-- hot module reloading in development mode
-- bundle splitting
-- universal rendering (run the same code in the server and the browser)
-- server-side development via Koa.js
-- plugin-based architecture (so you only include what you need in your browser bundles)
-- a curated set of plugins for data fetching, styling, etc. maintained by the Fusion.js team
-- plugins for error logging, security, etc.
-- bundle analysis tooling
+---
 
-If you want to know how Fusion.js compares to similar projects, see the [framework comparison page](/docs/learn-fusion/framework-comparison).
+### Step 1: Create a boilerplate app
 
----
+Creating a Fusion.js application is easy. In this tutorial, we will use [Yarn](https://code.fb.com/web/yarn-a-new-package-manager-for-javascript/),
+an open source package manager created at Facebook, to set up the Fusion.js scaffold for our application. Yarn replaces
```

Comment:
What are "existing workflows" referenced here?  Is this essentially saying that Yarn is essentially a better NPM?

Reply:
It's saying that yarn replaces the current existing workflow comprising of using npm to connect to the npm registry.